tools/hermetic_grunt/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# To build and run the hermetic grunt:
+# docker build -f tools/hermetic_grunt/Dockerfile -t uproxy-grunt . && \
+# docker run --rm -v $(pwd):/root/repository_root -it uproxy-grunt
+#
+# You have to run from the repository root. The output is written to the out/
+# directory.
+
+FROM ubuntu
```

Comment:
Sorry, you're right, I wasn't very clear :-) `ubuntu:trusty` sounds good to me for reasons of caching, good guess :-)